docs/reference/authentication.md

```diff
@@ -27,6 +27,17 @@ Web UI at Administration / Users to create users.  Users have an access key
 `AKIA...` and an associated secret access key.  These credentials are valid
 to log into the Web UI, or to authenticate programmatic requests to the API
 Server or the S3 Gateway.
+git
+#### lakeFS Invite
+
+If you were invited to lakeFS, Follow these steps to create and activate your account:
+
+1. Locate the email sent to you, (be aware it may be in your spam folder)
+1. Click on the link in the email, to be directed to the account activation page
+1. Set your account password
+1. Now you can login to the UI using your email and the password you created for your account in the previous step
+
+Note that email/password credentials should only be used for login via the UI.
```

Comment:
Please rephrase in a more positive mode: IIUC what this is saying is that you can only use email and password to login to the GUI.  So you should also say how to connect to everything else.

Also, I am not sure that this is correct: can I use email and password in `lakectl`?  Because that is also our UI.  If I *cannot* use email and password in `lakectl`, then this should be "GUI".

Reply:
Added the point that a set of access and secret keys should be generated.
Using the email/password is not supported for anything besides the GUI. (It will technically work though). Fixed to GUI.